core/zstd/inc/ZipZSTD.h

```diff
@@ -0,0 +1,22 @@
+// Author: Brian Bockelman March 2015
+
+/*************************************************************************
+ * Copyright (C) 1995-2017, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+// NOTE: the ROOT compression libraries aren't consistently written in C++; hence the
+// #ifdef's to avoid problems with C code.
```

Comment:
Is "the ROOT compression libraries aren't consistently written in C++" still the case?

Either way it's fine to have them `extern "C"`, we can probably just remove the comment.